### ISSUE TITLE
Redis passwords don't accept spaces

### ISSUE THREAD

**Issue by alexdesousa** [artemeff/exredis#51]
Hello,

I was having problems connecting to my Redis database, always receiving this error:

```erlang
** (EXIT from #PID<0.163.0>) {:connection_error, {:authentication_error, {:unexpected_data, {:ok, "-ERR wrong number of arguments for 'auth' command\r\n"}}}}
```

In my Elixir configuration I had:

```elixir
config :exredis,
  host: "localhost",
  port: 6379,
  password: "Password with spaces.",
  db: 1,
  reconnect: :no_reconnect,
  max_queue: :infinity
```

To make the library work with my password, I had to change the password field of the configuration to:

```elixir
(...)
  password: "\"Password with spaces.\"",
(...)
```

Tracing the bug, I got to the authentication functions in eredis. When I executed the following:
```erlang
{ok, C} = eredis:start_link("localhost", 6379, 1, "Password with spaces.")
```
I got:
```erlang
** exception exit: {connection_error,{authentication_error,{unexpected_data,{ok,<<"-ERR wrong number of arguments for 'auth' command\r\n">>}}}}
```
I already made a [pull request to eredis](https://github.com/wooga/eredis/pull/79) with a patch to this bug.

Anyway, thank you for the library.

TL;DR: At the moment, eredis is not accepting passwords with spaces and exredis inherits this problem. Earlier this day, I made a [pull request to eredis](https://github.com/wooga/eredis/pull/79) that fixes this bug in eredis, but I am waiting for approval.

**Comment by artemeff** [comment]
Thank you :+1: I'll update dependency when PR is merged.
